README.md

```diff
@@ -90,10 +90,9 @@ your project as explained below.
 
 To use Swift with Protocol buffers, you'll need:
 
-* A Swift 5.0 or later compiler (Xcode 10.2 or later).  Support is included for
-  the Swift Package Manager. The Swift protobuf project is being developed and
-  tested against the latest release version of Swift available from
-  [Swift.org](https://swift.org)
+* A Swift 5.6 or later compiler (Xcode 14.1 or later). The Swift protobuf project
```

Comment:
When I read "Swift x.y (Xcode x.y)", I tend to interpret those as being associated together, but that's not the case here since the minimum Xcode has a higher version than the overall Swift version we support, due to Apple factors.

Maybe rephrase this to be more explicit about the divergence, like this?

> A Swift 5.6 or later compiler (or, if building with Xcode, Xcode 14.1 or later as required by the App Store)

If there's a future update where the minimum NIO supported version is equal to or greater than the minimum supported Xcode version, then we can condense it back down to "A Swift 5.6 or later compiler (Xcode x.y)".

Reply:
Done.